poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
This is a hint to the next PR about updating `poetry` up to v2.0: 

Please see these release notes: https://python-poetry.org/blog/announcing-poetry-2.0.0/